sklearn/metrics/tests/test_classification.py

```diff
@@ -678,7 +678,7 @@ def test_matthews_corrcoef_multiclass():
     assert_almost_equal(mcc, 0.)
 
 
-@pytest.mark.parametrize('n_points', [100, 10000, 1000000])
```

Comment:
The last case was taking 6s on my laptop, which is too much for a test, and doesn't seem to add much compared to the other two.